tests/v0_tests/test_index.py

```diff
@@ -209,7 +219,11 @@ def test_create_custom_number_of_replicas(self):
         settings = {
             "number_of_replicas": intended_replicas,
         }
-        test_index_name = self.create_test_index(index_name=self.generic_test_index_name, settings_dict=settings)
+        test_index_name = self.create_test_index(
+            cloud_test_index_to_use=CloudTestIndex.basic_index,
```

Comment:
if this test is ignored, can you not set the cloud_test_index_to_use? As it doesn't make sense here.